docs/microservices.json

```diff
@@ -34,17 +34,13 @@
             "id": "draft-store",
             "name": "Draft Store",
             "spec": "https://hmcts.github.io/reform-api-docs/specs/draft-store.json",
-            "group": "Draft Store",
+            "group": "Platform Engineering",
+            "repository": "https://github.com/hmcts/draft-store",
             "dependencies": [
                 {
                     "id": "idam-s2s",
                     "hard": true,
                     "apis": []
-                },
-                {
-                    "id": "idam",
-                    "hard": true,
-                    "apis": []
```

Comment:
this should stay.

Reply:
I've not removed it, just renamed from idam to idam-s2s to match the idam definition now

Reply:
look again, you've renamed 'idam' to 'idam-web'